app.rb

```diff
@@ -11,11 +12,25 @@ class Mkbnb < Sinatra::Base
 
   get '/' do
     @users = User.all
-    
-    # @users.each { |user| p user.first_name } 
     erb :index
   end
 
+  get '/create_listing' do
+    erb(:new_listing)
+  end
+
+  post '/listings' do
+    Rooms.create!(
+      title: params[:title],
+      description: params[:description],
+      price_per_night:params[:price_per_night].to_f,
+      user_id: User.all.last.id
+    )
```

Comment:
we weren't sure if we should define a method within the `Room` class called `add_listing` which calls exactly this ActiveRecord `create!` method - it will be the exact same amount of lines/logic in the `app.rb` file if we did but might be cleaner ?